sdk/digitaltwins/Azure.DigitalTwins.Core/samples/DigitalTwinsClientSample/ComponentSamples.cs

```diff
@@ -106,7 +105,7 @@ public async Task RunSamplesAsync(DigitalTwinsClient client)
                 {
                     Metadata = new Component1Metadata { ModelId = componentModelId },
                     ComponentProp1 = "Component prop1 val",
-                    ComponentProp2 = "Component prop2 val",
+                    ComponentProp2 = 123,
```

Comment:
still needs a different new value than the DT prop

Reply:
not sure I understand, this is the value that we are setting to the twin's component's property. do you mean set different values for `BasicDigitalTwin` vs `CustomDigitalTwin`?

Reply:
My mistake. You need to change Prop2 type to something other than string too.

Reply:
Ah, got it. Sure, will do.